crates/sui-indexer-alt-graphql/src/api/types/checkpoint/mod.rs

```diff
@@ -212,8 +213,14 @@ impl CheckpointContents {
             return Ok(Some(Connection::new(false, false)));
         };
 
+        let available_range = AvailableRange::new(
```

Comment:
We may need a type here that holds the pipelines, that's something that `AvailableRange` does not need for its GraphQL implementation, and vice versa, this code doesn't care about `Scope`, so we should use different types for both cases.